Game.java

```diff
@@ -0,0 +1,70 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.Scanner;
+
+public class Game {
+	
+	enum Creatures {
+		Pacman,
+		Ghost,
+		Point,
+		Wall,
+		Null
+	}
+	
+	Creatures[][] board = new Creatures[30][30];
+	
+	public int getBoardDimensions(){
+		return board.length * 100 + board[0].length;
+	}
+	
+	public Creatures[][] setBoard() throws Exception{
+		// Change Board.txt path on another computer.
+		// Home path: E:/Docs/שלום/A android project 2018/Pacman/Board.txt
+		Scanner sc = new Scanner(new BufferedReader(new FileReader("E:/Docs/שלום/A android project 2018/Pacman/Board.txt")));

```

Comment:
You should read the file using relative path, not absolute one
(If I will take this code to my computer, it wont work...)